bin/docker/gen-dockerfile

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -eu
+
+# Defaults
+default_input=Dockerfile.beaver
+docker_dir=docker
+docker_build=build
+docker_dist=${DIST:-$(lsb_release -cs)}
+
+# Transforms a fragment (for now it just injects the DIST
```

Comment:
I think this reply from you was referring to the name of `Dockerfile.inc` and now github somehow merged it with this comment. You're still missing the closing `)`.